modules/host/host.go

```diff
@@ -343,18 +126,41 @@ type Host struct {
 	resourceLock sync.RWMutex
 }
 
+// checkUnlockHash will check that the host has an unlock hash. If the host
+// does not have an unlock hash, an attempt will be made to get an unlock hash
+// from the wallet. That may fail due to the wallet being locked, in which case
+// an error is returned.
+func (h *Host) checkUnlockHash() error {
```

Comment:
name seems a little weird, since "check" doesn't usually imply that the function will modify state.